homeassistant/components/airzone_cloud/strings.json

```
@@ -45,7 +45,7 @@
           "fan": "[%key:component::climate::entity_component::_::state::fan_only%]",
           "heat": "[%key:component::climate::entity_component::_::state::heat%]",
           "heat_cool": "[%key:component::climate::entity_component::_::state::heat_cool%]",
-          "stop": "Stop"
+          "stop": "[%key:common::state::stopped%]"
```

I think you should change this back since the mode actually appears to be called `Stop`

See [here](https://www.airzonecontrol.com/en/en/support/faqs/perche-non-e-presente-uno-stop-sullo-schermo-del-mio-termostato/)

```suggestion
          "stop": "Stop"
```